specification/chaos/resource-manager/Microsoft.Chaos/preview/2024-03-22-preview/privateAccesses.json

```diff
@@ -231,6 +231,55 @@
           }
         }
       },
+      "patch": {
+        "tags": [
+          "PrivateAccesses"
+        ],
+        "operationId": "PrivateAccesses_UpdateTags",
+        "description": "Update a private access tags",
+        "parameters": [
+          {
+            "$ref": "./types/common.json#/parameters/apiVersion"
+          },
+          {
+            "$ref": "./types/common.json#/parameters/subscriptionId"
+          },
+          {
+            "$ref": "./types/common.json#/parameters/resourceGroupName"
+          },
+          {
+            "$ref": "#/parameters/PrivateAccessName"
+          },
+          {
+            "name": "privateAccess",
```

Comment:
this shouldn't be the entire private access object, since we are not supporting the entire object. You should create a new payload type that only has tags in it, similar to experiment